tools/service_bundle.dtd.1

```diff
@@ -0,0 +1,1091 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this used by anything?  The only reason I can think that we would need a copy is to validate xml on a non-illumos machine.  Do we do that?

Reply:
On the MacOS:

```
xmllint --noout --path deps/eng/tools/ --dtdvalid deps/eng/tools/service_bundle.dtd.1 smf/manifests/kbmapi.xml
xmllint --noout --path deps/eng/tools/ --dtdvalid deps/eng/tools/service_bundle.dtd.1 smf/manifests/kbmtr.xml
```